src/test/java/com/teamscale/upload/NativeImageIT.java

```diff
@@ -242,19 +245,23 @@ public void printStackTraceForKnownErrorsOnlyWhenRequested() {
 			ProcessUtils.ProcessResult result = runUploader(
 					new Arguments().withUrl("https://localhost:" + MOCK_TEAMSCALE_PORT));
 			assertSoftlyThat(softly -> {
-				softly.assertThat(result.exitCode).describedAs("Stderr and stdout: " + result.stdoutAndStdErr)
+				softly.assertThat(result.exitCode).describedAs("Stderr and stdout: " + result.getOutputAndErrorOutput())
 						.isNotZero();
-				softly.assertThat(result.stdoutAndStdErr).contains("--stacktrace");
+				softly.assertThat(result.errorOutput).contains("--stacktrace");
 			});
 		}
+	}
 
+	@Disabled("This seems to cause the windows build to get stuck after the change to the ProcessBuilder API in TS-29179")
+	@Test
+	public void printStackTraceForKnownErrorsOnlyWhenRequested2() {
```

Comment:
This test seems to be the reason for the stuck windows build. I can reproduce this locally, i.e. the test does not terminate. This must have something to do with the refactoring of the ProcessUtils. On master I can successfully run the test.

Do you have any idea what could cause this?

Reply:
The ProcessBuilder API unfortunately also has some quirks. See code comments in ProccessUtils.